csharp/ql/src/utils/model-generator/internal/CaptureModels.qll

```diff
@@ -247,7 +247,7 @@ string captureSource(DataFlowTargetApi api) {
  * A TaintTracking Configuration used for tracking flow through APIs.
  * The sources are the parameters of the API and the fields of the enclosing type.
  *
- * This can be used to generate Sink summaries for APIs, if the API propgates a parameter (or enclosing type field)
+ * This can be used to generate Sink summaries for APIs, if the API propagates a parameter (or enclosing type field)
```

Comment:
This file and `java/ql/src/utils/model-generator/internal/CaptureModels.qll` need to be in sync, so you need to update them on the same PR (e.g. this PR).